sycl/test-e2e/Basic/submit_time.cpp

```diff
@@ -1,34 +1,34 @@
 // RUN: %{build} -o %t.out
-// RUN: %{run} %t.out
+// There is an issue with reported device time for the L0 backend, works only on
+// pvc for now. No such problems for other backends.
+// RUN: %if (!ext_oneapi_level_zero || gpu-intel-pvc) %{ %{run} %t.out %}
```

Comment:
Should we make it a `REQUIRES` instead? Seems like it would be better to just skip it. I don't think compile-only testing benefits us in E2E.

Reply:
I tried with REQUIRES at first, but for some reasons test is still executed if I do this. That's why changed to this approach.